force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -81,15 +99,26 @@ public with sharing class ClassesObjectsMethods {
      */
     public static String removeDuplicateWords(String sentence) {
         // Split the sentence into words
+        List<String> wordsFromSentence = sentence.split(' ');
 
         // Create a set to store the unique words
+        Set<String> uniqueWordSet = new Set<String>();
 
         // Add each word to the set
-
+        for (String word : wordsFromSentence){
+            if(!uniqueWordSet.contains(word)) {
+                uniqueWordSet.add(word);
+            }
+        }
         // Join the unique words back into a sentence
+        String newSentence = '';
+        for(String word1 : uniqueWordSet) {
+            newSentence += word1 + ' ';
+        }
 
+        newSentence = newSentence.substring(0, newSentence.length() - 1);
         // Return the sentence with unique words
-        return null; // Replace null with the variable you used to store the result
+        return newSentence; // Replace null with the variable you used to store the result
```

Comment:
While this works. It is probably much harder than it needs to be. You can pass a List directly to a set and skip all the iteration. And use the String.join method to create a new String. 
```
 public static String removeDuplicateWords(String sentence) {
        Set<String> uniqueWords = new Set<String>(sentence.split(' '));
        return String.join(uniqueWords, ' ');
    }
 ```

Reply:
Good point.